config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -5,7 +5,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.1
-    api-approved.kubernetes.io: "unapproved, experimental-only"
```

Comment:
Looks like unrelated change -> separate PR(?)

Reply:
is not a change... `make manifest` is deleting that line everytime 🤦🏽

Reply:
OK. It's just seems really hard for me to understand what is being autogenerated and what. Could you submit these `make manifests` related changes as a separate PR?

Reply:
I need to find a way to `make manifest` not to remove that line, but so far I haven't found documentation for it, but sure will be a following PR

Reply:
👍